modules/ingest-common/src/test/resources/rest-api-spec/test/ingest/200_default_pipeline.yml

```diff
@@ -0,0 +1,65 @@
+---
+teardown:
+  - do:
+      ingest.delete_pipeline:
+        id: "my_pipeline"
+        ignore: 404
+
+---
+"Test index with default pipeline":
```

Comment:
While having rest tests is fine to ensure the param is passed through the rest layer correctly, we should first have unit tests.